### ISSUE TITLE
Example from the tutorial Panics

### ISSUE THREAD

**Issue by OrioPrisco** [uiua-lang/uiua#499]
Executing the following example from the tutorial in the [pad ](https://www.uiua.org/pad) Does not output anything, and checking the console using firefox's dev tools shows that a panic happened.
```uiua
~ "git: github.com/uiua-lang/example-module" ~ Upscale
Upscale 3 [1_2 3_4]
```

```
panicked at C:\Users\Kai.Schmidt\.cargo\registry\src\index.crates.io-6f17d22bba15001f\leptos_reactive-0.6.9\src\runtime.rs:1278:13:
couldn't locate ResourceId(1v1) at Location { file: "C:\\Users\\Kai.Schmidt\\.cargo\\registry\\src\\index.crates.io-6f17d22bba15001f\\leptos_reactive-0.6.9\\src\\runtime.rs", line: 1280, col: 17 }

Stack:

__wbg_get_imports/imports.wbg.__wbg_new_abda76e883ba8a5f@https://www.uiua.org/site-5fa4282b570bec99.js:458:21
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[9142]:0x71c8fb
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[4184]:0x6254f9
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[7426]:0x6eeb20
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[667]:0x2509ce
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[254]:0xde4b1
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[271]:0x108961
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[397]:0x18f237
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[240]:0xa1ca9
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[5987]:0x6be610
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[370]:0x17b6ec
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[1235]:0x35de19
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[5193]:0x697c6a
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[4928]:0x658fdd
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[1760]:0x48a96a
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[3118]:0x5a8a3e
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[386]:0x188344
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[8314]:0x709aba
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[8108]:0x700c4e
__wbg_adapter_36@https://www.uiua.org/site-5fa4282b570bec99.js:224:10
real@https://www.uiua.org/site-5fa4282b570bec99.js:209:20
__wbg_get_imports/imports.wbg.__wbg_call_b3ca7c6051f9bec1/<@https://www.uiua.org/site-5fa4282b570bec99.js:1471:33
handleError@https://www.uiua.org/site-5fa4282b570bec99.js:253:18
__wbg_get_imports/imports.wbg.__wbg_call_b3ca7c6051f9bec1@https://www.uiua.org/site-5fa4282b570bec99.js:1470:63
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[6868]:0x6dd72d
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[1753]:0x488e2f
@https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:wasm-function[8152]:0x705bb2
__wbg_adapter_42@https://www.uiua.org/site-5fa4282b570bec99.js:232:10
real@https://www.uiua.org/site-5fa4282b570bec99.js:209:20


[site-5fa4282b570bec99.js:471:13](https://www.uiua.org/site-5fa4282b570bec99.js)
    __wbg_error_f851667af71bcfc6 https://www.uiua.org/site-5fa4282b570bec99.js:471
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:7457226
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:6444281
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:7269152
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:2427342
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:910513
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:1083745
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:1634871
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:662697
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:7071248
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:1554156
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:3530265
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:6913130
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:6655965
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:4761962
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:5933630
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:1606468
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:7379642
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:7343182
    __wbg_adapter_36 https://www.uiua.org/site-5fa4282b570bec99.js:224
    real https://www.uiua.org/site-5fa4282b570bec99.js:209
    __wbg_call_b3ca7c6051f9bec1 https://www.uiua.org/site-5fa4282b570bec99.js:1471
    handleError https://www.uiua.org/site-5fa4282b570bec99.js:253
    __wbg_call_b3ca7c6051f9bec1 https://www.uiua.org/site-5fa4282b570bec99.js:1470
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:7198509
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:4754991
    <anonymous> https://www.uiua.org/site-5fa4282b570bec99_bg.wasm:7363506
    __wbg_adapter_42 https://www.uiua.org/site-5fa4282b570bec99.js:232
    real https://www.uiua.org/site-5fa4282b570bec99.js:209
```
This example can be executed from the tutorial at https://www.uiua.org/tutorial/modules. It also seems to have a similar error the first time but works fine once the page is reloaded.
This error happens in both `Mozilla Firefox 126.0` and `Chromium 124.0.6367.201 snap` on my home computer, and on other browsers on various devices.

**Comment by OctoRocket** [comment]
I can confirm I'm having the same issue with the same code.

**Comment by kaikalii** [comment]
This should be fixed in 5565ade9c7e6a745a809.
I've deployed the change to the site
